Kickstarter-iOS/Tests/Views/ProjectPamphletContentViewControllerTests.swift

```diff
@@ -32,7 +32,7 @@ internal final class ProjectPamphletContentViewControllerTests: TestCase {
 
   override func tearDown() {
     AppEnvironment.popEnvironment()
-    UIView.setAnimationsEnabled(true)
+    UIView.setAnimationsEnabled(false)
```

Comment:
ok am gonna switch back to true for now and revisit updating the base class for the sake of time.